src/ott-analytics.js

```diff
@@ -237,7 +237,7 @@ class OttAnalytics extends BasePlugin {
         : MEDIA_TYPE,
       fileId: this._fileId,
       mediaId: this.config.entryId,
-      position: this.player.currentTime
+      position: this.player.isLive() ? this.player.currentTime - this.player.getStartTimeOfDvrWindow() : this.player.currentTime
```

Comment:
This will only give a sane result in case that start of dvr window is a fixed value. If window is sliding it will just be a static value from 0 to dvr window length.

Reply:
@OrenMe this is how it was before the live current time change (see https://github.com/kaltura/playkit-js/pull/559/files#diff-c04fed827979ddc81e94657cbfd42bf8f1dffd0224afd69253234a313e1f524bL272)

Reply:
you are right :-), thanks